libs/dao/src/main/java/com/akto/dto/billing/FeatureAccess.java

```diff
@@ -1,18 +1,36 @@
 package com.akto.dto.billing;
 
+import org.bson.codecs.pojo.annotations.BsonIgnore;
 import com.akto.dao.context.Context;
 
 public class FeatureAccess {
-    public boolean isGranted;
+    boolean isGranted;
     public static final String IS_GRANTED = "isGranted";
-    public int overageFirstDetected = -1;
+    int overageFirstDetected = -1;
     public static final String OVERAGE_FIRST_DETECTED = "overageFirstDetected";
```

Comment:
We can keep access modifiers of instance vars to be private if they have apt getters and setters